docs/docs/concepts/policies.mdx

```diff
@@ -2,5 +2,699 @@
+what a list of policies might look like:
+
+```yaml-rasa title="config.yml"
+...
+
+policies:
+  - name: FlowPolicy
+  - name: DocSearchPolicy
+```
+
+
+:::
+
+## Action Selection
+
+At every turn, each policy defined in your configuration gets a chance to
+predict a next action with a certain confidence level. A policy can also not predict an action.
+The policy that predicts with the highest confidence decides the assistant's next action.
+
+:::note Maximum number of predictions
+By default, your assistant can predict a maximum of 10 next actions
+after each user message. To update this value,
+you can set the environment variable `MAX_NUMBER_OF_PREDICTIONS`
+to the desired number of maximum predictions.
+
+:::
+
+
+### Flow Policy
+
+:::info New in 3.7
+
+The *Flow Policy* is part of Rasa's new 
+Conversational AI with Language Models (CALM) approach and available starting
+with version `3.7.0`.
+:::
+
+Rasa's Flow Policy is a state machine that allows you to manage and control your
+chatbot's conversational flows. It facilitates the integration of your business
+logic with machine learning capabilities.
+
+The Flow Policy oversees your bot's state, handles state transitions, and
+triggers new flows when needed. The "Dialogue Stack" represents all started but
+incomplete flows and refers to steps defined in the
+[flows specification](../concepts/flows.mdx).
+
+
+
+Using the Flow Policy, you can encapsulate your logic into flows, thereby
+guiding the conversation based on user input and system response. Other
+classifiers adjust the underlying state machine to handle edge cases.
+
+For more detailed information on defining flows, refer to the
+[flows specification](../concepts/flows.mdx).
+
+## Adding the FlowPolicy to your bot
+
+The Flow Policy can be integrated like [any other policy](../building-classic-assistants/policies.mdx). It
+involves two steps:
+
+1. Configure the `FlowPolicy` in your `config.yml` file:
+
+   ```yaml-rasa title="config.yml"
+   policies:
+   # - ...
+     - name: rasa.core.policies.flow_policy.FlowPolicy
+   # - ...
+   ```
+
+2. Add flows to your bot as per the [flows specification](../concepts/flows.mdx). This
+   step involves defining the states and transitions that your bot should
+   recognize and react to.
+
+Once added, the `FlowPolicy` will read all defined flows from your training data
+and apply them.
+
+## How does it work?
+
+The `FlowPolicy` employs a dialogue stack structure (Last In First Out) along with
+internal slots to manage your bot's state.
+
+### Managing the State
+
+Your bot's state is managed using a "dialogue stack". The dialogue stack stores the
+current position in each started flow. The stack follows a "last in, first out"
+sequence. The last started flow will be completed first, and then earlier
+inserted flows will continue. The dialogue stack is stored in the `dialogue_stack` slot.
+
+Here's an example flow and a conversation currently using it:
+
+```yaml title="flows.yml"
```

Comment:
ideally we change it like this to have the example validated
```suggestion
```rasa-yaml title="flows.yml"
```